### PR TITLE
Fixes gravity in new areas

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Creating a new area in lavaland or the asteroid or whatever would result in an area with 0 gravity. This fixes that.

### CODE REVIEW

File: code/game/objects/items/blueprints.dm [hunk 1]
```diff
@@ -216,6 +218,7 @@
 		A.setup(str)
 		A.contents += turfs
 		A.SetDynamicLighting()
+	A.has_gravity = old.has_gravity
```
Comment (RemieRichards) [inline code/game/objects/items/blueprints.dm]: rare, but what if you managed to remove every instance of an Area? just check old exists first please.

Comment (Jordie0608) [inline code/game/objects/items/blueprints.dm]: Why not just do `A.has_gravity = has_gravity(src)` ?

Comment (xxalpha) [inline code/game/objects/items/blueprints.dm]: This, isn't the proc supposed to handle that?

Comment (KorPhaeron) [inline code/game/objects/items/blueprints.dm]: has_gravity(src) would return whether the area currently has gravity (including from a generator), not whether the area innately has it (such as lavaland or centcomm)

Comment (Jordie0608) [inline code/game/objects/items/blueprints.dm]: Sans the gravity generator, your code has the exact same function as has_gravity()

File: code/game/objects/items/blueprints.dm [hunk 2]
```diff
@@ -193,13 +193,15 @@
...
 				return
+
 	var/list/turfs = res
 	var/str = trim(stripped_input(usr,"New area name:", "Blueprint Editing", "", MAX_NAME_LEN))
 	if(!str || !length(str)) //cancel
 		return
 	if(length(str) > 50)
 		usr << "<span class='warning'>The given name is too long.  The area remains undefined.</span>"
 		return
+	var/area/old = get_area(get_turf(src))
```
Comment (KazeEspada) [inline code/game/objects/items/blueprints.dm]: var/area/old = get_area(get_turf(src))
var/oldgravity = old.has_gravity

and then later....
A.has_gravity = oldgravity

**Pull request merged**